### PR TITLE
Bumps k8s deps for v1.19.0-rc.2

### PR DESCRIPTION
**Pull request by danehans**
Bumps k8s deps due to https://github.com/openshift/api/commit/0ba2c3658da6cade27a5066575cd8446eb03914c and required to bump openshift/api to latest commit.

/assign @knobunc @Miciah 
/cc @frobware @sgreene570

### CODE REVIEW

**Comment by danehans** [comment]
Failing due to https://github.com/kubernetes-sigs/controller-runtime/issues/1033

**Comment by danehans** [comment]
```
There are unexpected changes to the vendor tree following 'go mod vendor' and 'go mod tidy'. Please
run these commands locally and double-check the Git repository for unexpected changes which may
need to be committed.
make: *** [verify] Error 1
```

I ran `go mod vendor` and `go mod tidy` locally without any errors or additional vendor updates.

/retest

**Comment by danehans** [comment]
@Miciah PTAL at https://github.com/openshift/cluster-ingress-operator/pull/429#issuecomment-664047415. I'm running `go mod vendor` and `go mod tidy` without any issues but this continues to be a problem in CI. Do you have any suggestions on how to get around this issue?

**Comment by sgreene570** [comment]
@danehans what version of go are you using locally for these changes? 
CI appears to be using Go 1.13, perhaps using Go 1.14 (or 1.12?) could be causing these vendor issues?

File: go.mod [hunk 1]
```diff
@@ -4,30 +4,33 @@ go 1.13
...
 	github.com/davecgh/go-spew v1.1.1
 	github.com/ghodss/yaml v1.0.0
-	github.com/go-logr/logr v0.1.0
-	github.com/go-logr/zapr v0.1.1
-	github.com/google/go-cmp v0.3.1
+	github.com/go-logr/logr v0.2.0
+	github.com/go-logr/zapr v0.2.0
+	github.com/google/go-cmp v0.4.0
 	github.com/kevinburke/go-bindata v3.11.0+incompatible
 	github.com/openshift/api v0.0.0-20200609191024-dca637550e8c
```
Comment (sgreene570) [inline go.mod:18]: Should this PR also bump the vendored version of `openshift/api`?

**Comment by sgreene570** [comment]
When I pull these changes locally and run `go mod vendor`, I am seeing a diff. I am using go 1.14. 
@danehans are you running `go mod tidy` _after_ `go mod vendor`? Try flipping the order of those commands.

**Comment by Miciah** [comment]
I am seeing a diff too.  `git status` reports the following:

```
Changes not staged for commit:
	deleted:    vendor/github.com/googleapis/gnostic/OpenAPIv2/OpenAPIv2.go
	deleted:    vendor/github.com/googleapis/gnostic/OpenAPIv2/OpenAPIv2.pb.go
	deleted:    vendor/github.com/googleapis/gnostic/OpenAPIv2/OpenAPIv2.proto
	deleted:    vendor/github.com/googleapis/gnostic/OpenAPIv2/README.md
	deleted:    vendor/github.com/googleapis/gnostic/OpenAPIv2/openapi-2.0.json

Untracked files:
	vendor/github.com/googleapis/gnostic/openapiv2/
```

Could your file system or Go version be masking the change in case?  I'm using ext4fs and Go 1.13.8.  Try `rm -rf vendor/github.com/googleapis/gnostic/OpenAPIv2/` and `git mod vendor` again.

File: go.mod (hunk 1 above)
Comment (danehans) [reply go.mod:18]: `openshift/api` bump is not required for this PR. `openshift/api` will be bumped in https://github.com/openshift/cluster-ingress-operator/pull/427

**Comment by danehans** [comment]
@Miciah and @sgreene570 thanks for the assistance. It looks like I was hitting [this issue](https://stackoverflow.com/questions/6899582/i-change-the-capitalization-of-a-directory-and-git-doesnt-seem-to-pick-up-on-it). The verify job is now passing.

**Comment by danehans** [comment]
```
level=error msg="Cluster operator authentication Degraded is True with ConfigObservation_Error::IngressStateEndpoints_MissingSubsets::RouterCerts_NoRouterCertSecret: RouterCertsDegraded: secret/v4-0-config-system-router-certs -n openshift-authentication: could not be retrieved: secret \"v4-0-config-system-router-certs\" not found\nIngressStateEndpointsDegraded: No subsets found for the endpoints of oauth-server\nConfigObservationDegraded: secret \"v4-0-config-system-router-certs\" not found"
```

Artifacts indicate the control-plane is unavailable. The etcd operator for example:
```
             "name": "etcd",
                "resourceVersion": "11510",
                "selfLink": "/apis/config.openshift.io/v1/clusteroperators/etcd",
                "uid": "b1ee4682-a7e5-414b-8a61-518aed2d1909"
            },
            "spec": {},
            "status": {
                "conditions": [
                    {
                        "lastTransitionTime": "2020-07-27T18:32:18Z",
                        "message": "StaticPodsDegraded: pods \"etcd-ip-10-0-183-76.ec2.internal\" not found\nInstallerControllerDegraded: missing required resources: [configmaps: etcd-scripts,restore-etcd-pod, configmaps: config-1,etcd-metrics-proxy-client-ca-1,etcd-metrics-proxy-serving-ca-1,etcd-peer-client-ca-1,etcd-pod-1,etcd-serving-ca-1, secrets: etcd-all-peer-1,etcd-all-serving-1,etcd-all-serving-metrics-1]\nEnvVarControllerDegraded: at least three nodes are required to have a valid configuration\nScriptControllerDegraded: \"configmap/etcd-pod\": missing env var values\nRevisionControllerDegraded: configmaps \"etcd-pod\" not found",
                        "reason": "EnvVarController_Error::InstallerController_Error::RevisionController_ContentCreationError::ScriptController_Error::StaticPods_Error",
                        "status": "True",
                        "type": "Degraded"
                    },
                    {
                        "lastTransitionTime": "2020-07-27T18:30:25Z",
                        "message": "NodeInstallerProgressing: 1 nodes are at revision 0; 0 nodes have achieved new revision 1",
                        "reason": "NodeInstaller",
                        "status": "True",
                        "type": "Progressing"
                    },
                    {
                        "lastTransitionTime": "2020-07-27T18:30:18Z",
                        "message": "StaticPodsAvailable: 0 nodes are active; 1 nodes are at revision 0; 0 nodes have achieved new revision 1",
                        "reason": "StaticPods_ZeroNodesActive",
                        "status": "False",
                        "type": "Available"
                    },
```

/test e2e-aws

**Pull request merged**